src/configs/getAndroidConfig.ts

```diff
@@ -78,6 +99,15 @@ export function getAndroidConfig(theme: Theme): Platform {
     name: "compose/extra-colors",
     formatter: composeExtraColors,
   });
+  StyleDictionary.registerFormat({
+    name: "compose/core-colors",
+    formatter: composeCoreColors,
+  });
+  StyleDictionary.registerFormat({
+    name: "compose/icons",
+    formatter: composeIcons,
+  });
+  let className = _.upperFirst(_.camelCase(theme));
```

Comment:
```suggestion
  const className = _.upperFirst(_.camelCase(theme));
```

This reminds me that we should get linters running on this repository